test/cctest/test_inspector_socket_server.cc

```diff
@@ -318,8 +318,7 @@ class TestSocketServerDelegate : public SocketServerDelegate {
 
   void StartSession(int session_id, const std::string& target_id) override {
     session_id_ = session_id;
-    CHECK_NE(targets_.end(),
-             std::find(targets_.begin(), targets_.end(), target_id));
+    CHECK_NE(targets_.end(), std::ranges::find(targets_, target_id));
```

Comment:
We can use std::ranges::includes here in several other places as well

Reply:
@anonrig Maybe we can make that another PR? You want to do it after this one is merged?

I tried to keep the changes very small, just flipping to ranges, so that it is easy to review.